src/utils/thermodynamics_overrides.jl

```diff
@@ -0,0 +1,55 @@
+"""
+    TODO: Make sure this is consistent with the function in Thermodynamics.jl.
+"""
+# Once PR is merged in ClimaParams, import from there.
+# https://github.com/CliMA/ClimaParams.jl/pull/253
+const s_ref = 7
+
+function exner_function(thermo_params, ts)
+    R_d = TD.TP.R_d(thermo_params)
+    cp_d = TD.TP.cp_d(thermo_params)
+    p_0 = TD.TP.p_ref_theta(thermo_params)
+    p = TD.air_pressure(thermo_params, ts)
+    return (p / p_0)^(R_d / cp_d)
+end
+
+function theta_v(thermo_params, ts)
```

Comment:
Why don't we compute `Π` outside these functions, then make `theta_v` and others functions of `Π`, so we avoid recomputing it?